android/src/main/java/org/ligi/blexplorer/util/ManufacturerRecordParserFactory.java

```diff
@@ -101,7 +101,7 @@ public String getKeyDescriptor() {
             return "" + getCompanyIdentifierCode();
         }
 
-        public String getName(BluetoothDevice device) {
```

Comment:
Imho, this looks totally fine. But the commit message is a bit irritating. This PR seem to have been created by an automated process...